tests/test_indexer.py

```diff
@@ -332,6 +294,73 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def create_sample_bundle_created_event(self, bundle_key):
+        raise NotImplemented()
+
+    @staticmethod
```

Comment:
this (and it's descendents) should not be static methods.

Reply:
removed @staticmethod